zipline/finance/slippage.py

```diff
@@ -252,3 +349,124 @@ def process_order(self, data, order):
             price + (self.spread / 2.0 * order.direction),
             order.amount
         )
+
+
+class FixedEquitySlippage(FixedSlippageBase, EquitySlippageModel):
+    """
+    Model slippage for equities as a fixed spread.
+
+    Parameters
+    ----------
+    spread : float, optional
+        spread / 2 will be added to buys and subtracted from sells.
+    """
+    pass
+
+
+class FixedFutureSlippage(FixedSlippageBase, FutureSlippageModel):
+    """
+    Model slippage for futures as a fixed spread.
+
+    Parameters
+    ----------
+    spread : float, optional
+        spread / 2 will be added to buys and subtracted from sells.
+    """
+    pass
+
+
+class WithWindowData(object):
+    """
+    Mixin class for slippage models requiring average daily volume and
+    annualized daily volatility.
+    """
+
+    def __init__(self):
+        super(WithWindowData, self).__init__()
+        self._window_data_cache = ExpiringCache()
+
+    def _get_window_data(self, data, asset, window_length):
+        """
+        Internal utility method to return the trailing 20-day mean volume and
```

Comment:
We can probably remove the "20-day" in the doc now that it is parameterized and add it to the doc's parameter list.